### DISCUSSION TITLE
Using X_test and y_test for training CNN model

### DISCUSSION THREAD

**Discussion by yahayakayode** [community/community#41109] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi, 
Could someone explain to me why X_test and y_test could be used for model validation in the line of code below
history = model.fit(X_train, y_train, epochs=12, validation_data=(X_test, y_test), shuffle=True)

Thank.
